sql-cli/pyproject.toml

```
@@ -64,6 +64,6 @@ strict_equality = true
 show_error_codes = true
 show_error_context = true
 warn_redundant_casts = true
-warn_return_any = true
+#warn_return_any = true
```

```suggestion
warn_return_any = true
```
This is unrelated. I had issues with that too, but I would keep the discussion about mypy out of this PR.

---

I suggest we address this in a separate PR, there are more urging matters